web/src/lib/components/asset-viewer/panorama-viewer.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { fade } from 'svelte/transition';
+  import LoadingSpinner from '../shared-components/loading-spinner.svelte';
+  import { api, AssetResponseDto } from '@api';
+  import View360, { EquirectProjection } from '@egjs/svelte-view360';
+  import './panorama-viewer.css';
+
+  export let asset: AssetResponseDto;
+  export let publicSharedKey = '';
+
+  let assetData = '';
+  let errorMessage = '';
+
+  const loadAssetData = async () => {
+    try {
+      const { data } = await api.assetApi.serveFile(
+        { id: asset.id, isThumb: false, isWeb: false, key: publicSharedKey },
+        { responseType: 'blob' },
+      );
+
+      if (data instanceof Blob) {
+        assetData = URL.createObjectURL(data);
```

Comment:
assetData is a data url, right? Can we name this variable something like `fileUrl`, `dataUrl` instead?

Reply:
Sure, renamed it, thanks :) 
to be honest, I reused naming from photo-viewer component: https://github.com/immich-app/immich/blob/8a3ab5be3e2628a5c777876eab3dc23ac519f2e6/web/src/lib/components/asset-viewer/photo-viewer.svelte#L66

Reply:
Ah makes sense. That was before my time 😛.